tools/virtualization/installing-libvirt-and-virt-install-on-fedora-linux.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Installing libvirt and virt-install on Fedora Linux  
+subsection: virtualization  
+order: 3  
+---
+
+# Installing libvirt and virt-install on Fedora Linux
```

Comment:
I'd prefer moving the Libvirt description from `about` document here.

Reply:
Yep, makes sense.

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/6d697de92b7a4aa5f34c575fb3dfc18186b85900